src/org/thoughtcrime/securesms/notifications/NotificationCenter.java

```diff
@@ -320,7 +333,7 @@ public void addNotification(int chatId, int msgId) {
             NotificationPrivacyPreference privacy = Prefs.getNotificationPrivacy(context);
 
             DcMsg dcMsg = dcContext.getMsg(msgId);
-            String line = privacy.isDisplayMessage()? dcMsg.getSummarytext(100) : context.getString(R.string.notify_new_message);
+            String line = privacy.isDisplayMessage()? dcMsg.getSummarytext(2000) : context.getString(R.string.notify_new_message);
```

Comment:
This had no effect on my device. There was still only a single line of each message shown. I guess this has something to do with the "create a tiny inbox" part further down.

WhatsApp posts
- a notification group for each chat, showing each message as a single line. 
- It can be expanded to show them as separate notifications. 
- And they can each be expanded to show the whole message content.

Reply:
yip - but i have no idea how to make this expanding. we're using this inbox-style, https://itnext.io/android-inboxstyle-notification-as-deep-as-possible-4d74c0c725f1, however maybe better discuss this in #1415

wrt the longer test, i think 100 characters is already too small for wider devices, so i'd leave this in. also, systems may make use of the whole text (screen reader, whatever)

Reply:
But it was possible to expand notifications (at least if it was a single chat) on 1.8.1, do you still know how this was done?